ui/package.json

```diff
@@ -70,7 +70,7 @@
     "react-router-dom": "^5.1.2",
     "react-select": "^3.0.8",
     "react-spring": "8.0.27",
-    "react-switch": "^4.1.0",
+    "react-switch": "^6",
```

Comment:
Confirmed that `Toggle` still works as expected (our only usage of react-switch)

Reply:
(upgraded to 6 because version 4 and 5 list react 16 as a peer dependency)

Reply:
this apparently broke a few e2e tests, so I reverted it.  Our existing version 4.1.0 `Toggle`s continue to work in React 17, regardless of the peer dependency claim.